drop-in/src/main/java/com/adyen/checkout/dropin/DropIn.kt

```diff
@@ -62,91 +71,85 @@ object DropIn {
     }
 
     /**
-     * Starts the checkout flow to be handled by the Drop-in solution.
-     * Make sure you have [DropInService] set up before calling this.
-     * Call [registerForDropInResult] to create a launcher when initializing your Activity.
+     * Starts the checkout flow to be handled by the Drop-in solution. With this solution your backend only needs to
+     * integrate the /sessions endpoint to start the checkout flow.
      *
-     * You will receive the Drop-in result in the [DropInCallback] parameter specified when
-     * calling [registerForDropInResult].
+     * Call [registerForDropInResult] to create a launcher when initializing your Activity or Fragment and receive the
+     * final result of Drop-in.
      *
-     * 3 states can occur from this operation:
-     * - Cancelled by user: the user dismissed the Drop-in before it has completed.
-     * - Error: a [DropInServiceResult.Error] was returned in the [DropInService], or an error
-     * has occurred.
-     * - Finished: a [DropInServiceResult.Finished] was returned in the [DropInService].
+     * Use [dropInConfiguration] to configure Drop-in and the components that will be loaded inside it.
      *
-     * You should always handle the cases of cancellation and error in [DropInCallback.onDropInResult].
+     * Optionally, you can extend [SessionDropInService] with your own implementation and add it to your manifest file.
+     * This allows you to interact with Drop-in, and take over the checkout flow.
      *
      * @param context The context to start the Checkout flow with.
      * @param dropInLauncher A launcher to start Drop-in, obtained with [registerForDropInResult].
-     * @param paymentMethodsApiResponse The result from the paymentMethods/ endpoint.
+     * @param checkoutSession The result from the /sessions endpoint. Use [CheckoutSessionProvider] to create this.
```

Comment:
Shouldn't `/sessions` be `/sessions/{id}/setup`?

Reply:
`/sessions/{id}/setup` is a checkout shopper endpoint, they don't call it. They call `/sessions` and pass the response to `CheckoutSessionProvider` then we call `/sessions/{id}/setup`.

Does this make it any clearer?
`@param checkoutSession The result from the /sessions endpoint passed onto [CheckoutSessionProvider] to create this object.`

Reply:
Ohh I wasn't aware of that endpoint. Makes sense now